packages/amplify-prompts/src/progressbars/progressbar.ts

```diff
@@ -38,7 +38,7 @@ export type ItemPayload = {
     LogicalResourceId: string,
     ResourceType: string,
     ResourceStatus: string,
-    Timestamp: string,
```

Comment:
I believe its only use is to describe what the data is sent to be printed in the console:

for example:
deploying...
| resourceId | type | "DEPLOYMENT COMPLETE" | timestamp
| resourceId | type | "DEPLOYMENT COMPLETE" | timestamp
| resourceId | type | "CREATING_IN_PROGRESS" | timestamp

This is internal to the CLI, and only being used in one other package: amplify-provider-awscloudformation , which periodically provides status updates to the printer

I am curious to know more about how we are detecting version bump changes; since we export/import between packages, this may be an issue if packages are dependent on exported types. For example, the data team

Reply:
@sobolk I don't think we need to do so, the type is less restrictive.

@awsluja
Lerna looks at the conventional commits to determine which packages were changed and the type of change determined by the git commit message prefix (`feat:` is minor, `chore:` is a patch). Lerna bumps all relevant packages in the dependency tree during the release process.

Reply:
@jhockett on assignment yes we are more permissive, but on read we become ambiguous.

Would this work after change or not?

```
function foo(timestamp: string) {}

function bar(item: ItemPayload) {
  foo(item.Timestamp); // does this take date?
}
```

Reply:
Great catch, I missed that. I removed the need to change `Timestamp` from a `string` in this exported type.